core/domain/question_services.py

```diff
@@ -155,9 +155,15 @@ def get_questions_by_skill_ids(total_question_count, skill_ids):
             total_question_count questions linked to each given skill id.
             question count per skill will be total_question_count divided by
             length of skill_ids, and it will be rounded up if not evenly
-            divisable. The order of questions will follow the order of given
+            divisible. The order of questions will follow the order of given
```

Comment:
@sophiewu6, you missed this: 

> This description is good except for one thing. Please explain what happens if there are insufficient questions for a different skill.

Otherwise it looks good. This can be merged after you've fixed this, reverted package-lock.json, and the issue [here](https://github.com/oppia/oppia/pull/6850#discussion_r294017800) is filed. Thanks!

Reply:
Issue has been filed (#6930 )

Reply:
Done.